### PR TITLE
Add attribute summary in UG

### LINKED ISSUE #193: [PE-D][Tester B] Checking for description duplication is case-sensitive
**Command Involved**
meeting-add

**Observation:**
The checking for description is case-sensitive. If the user change the description of a meeting **at same time** from lower case letters to upper case letters and add again,  it is allowed.

**Steps to reproduce:**

￼

**Rationale behind:**
Since description are only texts to describe the meeting, if their only difference is the letter cases, they are most likely to convey the same meaning. Therefore, it would be more reasonable to consider descriptions with the only difference in letter cases the same.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: Ella-e/ped#4

### LINKED ISSUE #198: [PE-D][Tester B] Able to add multiple tags with same word but different letter cases
**Command Involved**
add

**Observation:**
The content for the tag is case-sensitive and multiple tags with same word but different letter cases are considered different tags.

**Steps to reproduce:**
￼

**Rationale behind:**
Since the case difference doesn't affect the meaning of the tag, would it be more reasonable to prevent multiple tags with same word but different letter cases to be added to the system?

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: Ella-e/ped#7

### PR DESCRIPTION
**Pull request by iynixil**
- Added attribute summary in UG (above `Command summary`)
- Removed `Restriction on Faculty values` under `Known Issues` in UG (moved to a table below attribute summary)
- Link valid faculty values to subheading of table

Part of #207, resolves #198, resolves #193

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -439,25 +439,45 @@ Furthermore, certain edits can cause the StaffConnect to behave in unexpected wa
...
+Module | m/ | Case-insensitive.<br>Valid module consisting of 2-4 letters, followed by exactly 4 numeric digits, with a suffix that is at most 1 character long. | `gess1025`, `hsi1000`, `cs2103t`
+Faculty | f/ | Case-insensitive.<br>Restricted set of values (refer to [valid faculty values](#valid-faculty-values) below). | `soc`, `biz`, `School of Business`
+Venue | v/ | Only alphanumeric characters allowed. | `belobog avenue`, `COM4-02-33`, `LT21`, `Kent Ridge Vale, Tulip Street, #12-34`
+Tag | t/ | Only alphanumeric characters allowed. | `tutor`, `professor`, `BestProf`, `Number1TA`
+Availability | a/ | Valid format of `day start-time end-time`.<br>1. `day` should be a valid day of week: `Monday`, `mon`, `Tuesday`, `tue`, `tues`, `Wednesday`, `wednes`, `wed`, `Thursday`, `thurs`, `thur`, `thu`, `Friday`, `fri`, `Saturday`, `satur`, `sat`, `Sunday`, `sun`.<br>2. `day` is case-insensitive.<br>3. `start-time` and `end-time` should be in the time format of `HH:mm` where `HH` is in 24 hours and `mm` are minutes 00-60. | `mon 13:00 14:00`, `monday 13:00 14:00`, `tues 14:00 21:00`
```
Comment (whitesnowx) [inline docs/UserGuide.md]: `minutes 00-60` to `minutes 00-59` --> 60 is an invalid min time

**Review by whitesnowx — COMMENTED** [review]
nitt

File: docs/UserGuide.md (hunk 1 above)
Comment (iynixil) [reply docs/UserGuide.md]: fixed!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -439,25 +439,45 @@ Furthermore, certain edits can cause the StaffConnect to behave in unexpected wa
...
+Meeting Description | d/ | Only alphanumeric characters allowed. Spaces are only allowed between alphanumeric characters. | `Meet for finals`, `Midterm revision`
+Meeting Start Time | s/ | Valid date and time format.<br>1. Valid date formats: `yyyy-MM-dd`, `yyyy-M-d`, `dd-MM-yyyy`, `yyyy-MM-d`, `d-MM-yyyy`, `d-M-yyyy`, `dd-M-yyyy`, `d/MM/yyyy`, `d-M-yyyy`, `dd-M-yyyy`, `dd/MM/yyyy`, `yyyy/MM/dd`, `yyyy/MM/d`, `yyyy/M/dd`,  `yyyy/M/d`<br>where `yyyy` is a 4-digit year (0000-9999), `M` is a single digit month (1-9), `MM` is a valid month (01-12), `d` is a single digit day, `dd` is a valid day (01-31)<br>2. Valid time formats: `HH:mm`, `H:mm`, `HHmm`<br>where `H` is a single digit hour (1-9), `HH` is a valid 24-hour (00-23), `mm` are valid minutes (00-59). | `2002-11-02 19:00`, `1-12-2022 9:00`, `2024/1/1 0000`
```
Comment (whitesnowx) [inline docs/UserGuide.md]: Availabilitity: 
> `HH` is in 24 hours

... `(xx-xx)`

Meeting Start Time
> `d` is a single digit day

...  `(x-x)`

**Review by whitesnowx — APPROVED** [review]
lgtm :)

File: docs/UserGuide.md (hunk 2 above)
Comment (iynixil) [reply docs/UserGuide.md]: fixed!

**Pull request merged**